packages/modules/src/firebase/checkout.ts

```diff
@@ -0,0 +1,49 @@
+import type { Request, Response } from 'firebase-functions';
+import { ValidateFunction } from 'ajv';
+import {
+  ajv,
+  sendRequestError,
+} from './ajv';
+import fixItems from './functions-checkout/fixItem';
```

Comment:
Nome de arquivo (quase) sempre kebab-case:

```suggestion
import fixItems from './checkout/fix-items';
```